test/test_command.py

```diff
@@ -258,39 +258,83 @@ def setUp(self):
 
     @patch('subprocess.check_output')
     @patch('subprocess.check_call')
-    def test_remove_extra_packages_no_discrepancies(self, mock_check_call, mock_check_output):
+    def test_determine_extra_packages_no_discrepancies(self, mock_check_call, mock_check_output):
         mock_check_output.return_value = 'mock==1.2\nDjango==1.7\nnose==1.3\n'
         packages = _create_packages()
 
-        self.command.remove_extra_packages(packages)
+        self.assertEqual((), self.command.determine_extra_packages(packages))
```

Comment:
For these tests, I'd prefer to assign the result to a variable and assert on that, with a blank line between the method call and the assert. I only do asserts on the function call itself for very simple tests with a single assert.